source/includes/linux/common-installation.rst

```diff
@@ -209,7 +209,7 @@ operating systems using RPM, DEB, or binary:
 
 The ``.deb`` or ``.rpm`` packages install the following 
 `systemd <https://www.freedesktop.org/wiki/Software/systemd/>`__ service file to 
-``/etc/systemd/system/minio.service``. For binary installations, create this
+``/lib/systemd/system/minio.service``. For binary installations, create this
```

Comment:
This explains why my SNSD .deb installs never create this file in /etc. 💡 

But my /etc one is respected, are there rules about which one wins if they both exist?